### PR TITLE
Split measurements strings up; issue #1811

### LINKED ISSUE #1811: Script to convert existing strings to new split up strings
The changes in https://github.com/mozilla/testpilot/pull/1786 and https://github.com/mozilla/testpilot/pull/1782 are helpful and appreciated (thanks, Chuck!).  Paragraphs are not a deal breaker for L10n, but multiple paragraphs should be broken up (as in the PRs -- the bulleted lists are a great example).  Any time we can keep HTML out of the localized strings, we should.

So, I'd like to land those PRs, but I don't want to ask the localizers who just spent all that time localizing to essentially do it all again in smaller chunks.  This bug is to look into converting at least the main parts of the .ftl files to use the new strings.  

/cc @mathjazz for any ideas since he already parses these files
/cc @flodolo for L10n interest

### PR DESCRIPTION
**Pull request by clouserw**
This closes #1811 and #1960 .  No change to formatting and no need to re-localize for the vast majority of strings.  The Cliqz measurements text was inconsistent with the others though, so it *does* change format (see screenshot) and we *do* lose the localizations (because simply joining a list with commas probably doesn't work in all languages).

one
two

/cc @chuckharmston for review.  Probably not worth spending too much time here until we get sign off from legal on the Cliqz change.

/cc @flodolo for comments and a heads up on the lost localizations.  Note that we'll need the Cliqz text in German very soon after this lands.  I'd like to land this ASAP as it'll likely bitrot quickly with the changes from pontoon coming in.

/cc @eleemoz if you look at this one file ( https://github.com/mozilla/testpilot/pull/2048/files#diff-7a2cd8cd266aceea79b69d17d01fc63d ) you can see how I reformatted the Cliqz text from bullet points to comma-separated.  Any concerns with that?

Thanks!

### CODE REVIEW

**Comment by clouserw** [comment]
This PR also doesn't extract new strings (this would include the new header and the cliqz rearrangement).  I didn't see how to do that in the docs, but if someone knows I can roll that into the PR. Thanks!

**Comment by chuckharmston** [comment]
@clouserw you have to manually copy it from the build directory, see #1781.

File: content-src/experiments/universal-search.yaml
Comment (chuckharmston) [inline content-src/experiments/universal-search.yaml:96]: Looks like your editor stripped these trailing spaces. I _think_ they don't matter? Worth double-checking.

File: frontend/src/app/containers/ExperimentPage.js
Comment (chuckharmston) [inline frontend/src/app/containers/ExperimentPage.js:388]: This code is amazing. A real pro must have written it. 👌

**Review by chuckharmston — APPROVED** [review]
Code looks great! r+ pending:

1. Copy the generated experiments.ftl into the right place.
1. Approval from legal.
1. Approval from @flodolo.
1. Cliqz-related changes.

**Comment by flodolo** [comment]
> /cc @flodolo for comments and a heads up on the lost localizations. Note that we'll need the Cliqz text in German very soon after this lands. I'd like to land this ASAP as it'll likely bitrot quickly with the changes from pontoon coming in.

I agree 😭 Can we try to make some extra time for localization this round? I know we tag on Thursday, but maybe we can cherry pick updates later, given the circumstances.

About the patch: it's great to finally see so much HTML cleaned up. As far as I can tell there are only 5 new strings: experimentMeasurementIntro and the 4 split up Cliqz. Is that correct? As far as I can tell, the first 2 strings can be literally copied and pasted from the previous version, minus markup.

As soon as this merges, I'll send out an email to the mailing list, also to double check the *measurements* strings.

**Comment by ghost** [comment]
@flodolo 
I can try to pick critical l10n after the tag.  Can you guarantee we can get German done?  Otherwise I'll not land until next sprint.

You are correct, there are only 5 new strings that need L10n.  There are about 20 more, but those were the ones I moved around with the script so they don't need L10n (but, technically they are new, so if something is weird, just wanted to point that out).  The experimentMeasurementIntro is already copy/pasted from the old file.  The Cliqz ones are all blank.

Edit: actually, I guess experimentMeasurementIntro is not copy/pasted.  I forgot it's a different file

File: frontend/src/app/containers/ExperimentPage.js
Comment (ghost) [reply frontend/src/app/containers/ExperimentPage.js:388]: 🗡  (that was me typing a grinning face, but it autocompleted to a dagger so gonna go with that)

File: content-src/experiments/universal-search.yaml
Comment (ghost) [reply content-src/experiments/universal-search.yaml:96]: They render correctly for me

**Comment by flodolo** [comment]
If landing this for next sprint, hopefully at the very beginning, is an option, it would be safer. 
Our main German localizer is still very active, but mostly in the week-ends.

**Comment by flodolo** [comment]
Got green light from German localizers

**Comment by koehlermichael** [comment]
The following is probably the list of German Cliqz-related strings split up properly:

Das Cliqz-Test-Pilot-Experiment wird durch Zusammenarbeit zwischen der Cliqz GmbH und Mozilla entwickelt. Cliqz schützt Ihre Privatsphäre und sammelt keine sensiblen Daten, mit denen Benutzerprofile erstellt werden können. Die Anonymisierung dieser Daten ist allerdings schwierig, und einzelne Personen können möglicherweise dennoch identifiziert werden.

Die Cliqz GmbH sammelt Daten über Ihre Such- und Surfaktivitäten, darunter in die Adressleiste eingegebenen Text, Suchanfragen, die Sie an bestimmte Suchmaschinen schicken, sowie Daten über Ihre Interaktion mit Webseiten, wie Mausbewegungen, Blättern und dort verbrachte Zeit.

Die Cliqz GmbH sammelt Telemetriedaten, darunter Ihre Interaktionen mit bestimmten Bereichen und Schaltflächen in Cliqz. Diese Daten werden mit einem eindeutigen Identifikator verknüpft, damit die Cliqz GmbH die Leistung im Laufe der Zeit nachvollziehen kann.

Mozilla sammelt Telemetriedaten, darunter Anzahl der Besuche auf Suchmaschinenseiten und welche Suchmaschinen Sie nutzen, sowie Cliqz’ eindeutigen Identifikator, mit dem Mozilla Korrelationen zwischen den Telemetriesystemen von Firefox und Cliqz suchen kann.

File: locales/cs/experiments.ftl
Comment (pdehaan) [inline locales/cs/experiments.ftl:24]: Did we lose the cliqz measurements?

Comment (chuckharmston) [reply locales/cs/experiments.ftl:24]: Yeah, but that's known. Cliqz formerly broke the normalized format we had, and there isn't a safe way to preserve the work (since, as @clouserw has noted, just joining with commas won't work in some languages.

**Comment by ghost** [comment]
Just to give an update here:  There is a discussion happening within the legal team about the ordering change I did on the Cliqz page.  If cliqz had been consistent with all the other experiments we'd have landed this already!  Aiming for start of next sprint now. :)

File: frontend/src/app/containers/ExperimentPage.js
Comment (pdehaan) [inline frontend/src/app/containers/ExperimentPage.js]: This looks like a typo. My eyes are telling me that the `<p>` tag from 387 above is closed on 388, so this looks like a lingering `href="/privacy">`.

File: frontend/src/app/containers/ExperimentPage.js
Comment (pdehaan) [inline frontend/src/app/containers/ExperimentPage.js]: Same as comment above. This looks like a lingering `href="/privacy">`

**Comment by jaredhirsch** [comment]
@chuckharmston trying out assigning PRs to lead reviewers, so we can see who owns what in the PR queue

**Comment by chuckharmston** [comment]
> @chuckharmston trying out assigning PRs to lead reviewers, so we can see who owns what in the PR queue

I've r+ed, action is on Wil to finish with legal concerns.

**Comment by ghost** [comment]
Ok, latest screenshot:

three

**Comment by ghost** [comment]
This is ready to land.  I'm waiting until we tag stage because cherry-picking after this will be harder since it touches so much.

**Comment by jaredhirsch** [comment]
Release has been tagged: https://github.com/mozilla/testpilot/releases/tag/2017-01-26-1

Merging.

**Comment by flodolo** [comment]
Note (also to self) for next time: the fact that the PR merges correctly is not enough.

For example we lost Albanian translations, because they were done between merge and the time the patch was prepared.
https://github.com/mozilla/testpilot/pull/2048/files#diff-29edbe1dde152efe9e8fa257de7c4b69

Localizer is already fixing them in Pontoon, no further action needed here.

**Comment by pdehaan** [comment]
We may have possibly missed the no-more-404s.yaml file.
Ref: _"TypeError: G.map is not a function" when viewing No More 404s page_ #2098

**Pull request merged**
